### PR TITLE
<Issue #42> 401 Error Screen

### PR DESCRIPTION
**Pull request by ZumariaStudios**
Yay 401 page! Let me know if I missed something / needs changes.

### CODE REVIEW

**Comment by ZumariaStudios** [comment]
#42

File: site_ifoundadog/templates/401.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% block title %}404{% endblock %}
```
Comment (cmhedrick) [inline site_ifoundadog/templates/401.html]: could you change the number from "404" to "401"?

**Review by cmhedrick — CHANGES_REQUESTED** [review]
There's a file named ".gitignore", could you add the following line to it:
.vs/

This should hopefully remove all the VS (visual studio) files from the commit. We just want to make sure that we don't add any IDE files to the repo.

If this doesn't work we can merge the changes, and I can fix the files.

Cheers!

File: site_ifoundadog/templates/401.html (hunk 1 above)
Comment (ZumariaStudios) [reply site_ifoundadog/templates/401.html]: Thanks for looking at that Chris, I completed these. Please let me know if there's anything else!

**Review by cmhedrick — APPROVED** [review]
Looks good

**Pull request merged**